src/config/southsudan/layers.json

```diff
@@ -0,0 +1,1200 @@
+      { "value": 110, "label": "110-120%", "color": "#00ff00" },
+      { "value": 120, "label": "120-130%", "color": "#00c900" },
+      { "value": 130, "label": "130-140%", "color": "#009600" },
+      { "value": 140, "label": "140-150%", "color": "#006400" },
+      { "value": 150, "label": "> 150%", "color": "#003200" }
+    ]
+  },
+  "lst_amplitude": {
+    "title": "Land Surface Temperature - 10-day Amplitude (MODIS)",
+    "type": "wms",
+    "server_layer_name": "myd11a2_taa_dekad",
+    "additional_query_params": {
+      "styles": "taa_21_1_48"
+    },
+    "base_url": "https://api.earthobservation.vam.wfp.org/ows/",
+    "wcsConfig": {
+      "scale": 0.02,
+      "offset": 0,
+      "pixelResolution": 64
+    },
+    "date_interval": "days",
+    "opacity": 0.7,
+    "legend_text": "Land Surface Temperature Amplitude refers to the difference between the maximum and minimum temperature in degrees Celsius",
+    "legend": [
+      { "value": 1, "label": "< 1C", "color": "#05284b" },
+      { "value": 50, "label": "1C to 2C", "color": "#0031ac" },
+      { "value": 100, "label": "2C to 3C", "color": "#0564c3" },
+      { "value": 150, "label": "3C to 4C", "color": "#1d8dbe" },
+      { "value": 200, "label": "4C to 5C", "color": "#57bec1" },
+      { "value": 250, "label": "5C to 6C", "color": "#004231" },
+      { "value": 300, "label": "6C to 8C", "color": "#006b52" },
+      { "value": 400, "label": "8C to 10C", "color": "#64a77e" },
+      { "value": 500, "label": "10C to 12C", "color": "#86cb66" },
+      { "value": 600, "label": "12C to 14C", "color": "#ccea83" },
+      { "value": 700, "label": "14C to 16C", "color": "#dcf1b2" },
+      { "value": 800, "label": "16C to 18C", "color": "#ffffbf" },
+      { "value": 900, "label": "18C to 20C", "color": "#ffeebd" },
+      { "value": 1000, "label": "20C to 24C", "color": "#ffec81" },
+      { "value": 1200, "label": "24C to 28C", "color": "#fed380" },
+      { "value": 1400, "label": "28C to 32C", "color": "#fec754" },
+      { "value": 1600, "label": "32C to 36C", "color": "#f5af28" },
+      { "value": 1800, "label": "36C to 40C", "color": "#d79b0b" },
+      { "value": 2000, "label": "40C to 44C", "color": "#d5a45f" },
+      { "value": 2200, "label": "44C to 48C", "color": "#bf7f2f" },
+      { "value": 2400, "label": "> +48C", "color": "#aa5a00" }
+    ]
+  },
+  "lst_daytime": {
+    "title": "Daytime Land Surface Temperature - 10-day (MODIS)",
+    "type": "wms",
+    "server_layer_name": "myd11a2_txa_dekad",
+    "additional_query_params": {
+      "styles": "tda_42_n24_70"
+    },
+    "base_url": "https://api.earthobservation.vam.wfp.org/ows/",
+    "date_interval": "days",
+    "wcsConfig": {
+      "scale": 0.02,
+      "offset": -273,
+      "pixelResolution": 64
+    },
+    "opacity": 0.7,
+    "legend_text": "Daytime land surface temperature",
+    "legend": [
+      { "value": 0, "label": "0C - 2C", "color": "#1d2e83" },
```

Comment:
In general, since the colors are gradual, I think we should limit the number of values in the legend. In this case in particular, le's try to limit to 5-10 values top.

It's also worth doing that in a few other legends as well imho.

<img width="358" alt="Screen Shot 2022-10-03 at 12 14 04 PM" src="https://user-images.githubusercontent.com/16843267/193553678-444246fc-b983-4f7e-a12f-103e314c7657.png">

Reply:
yup, working to refine this